ethereum/env/.env.snaxchain.testnet

```diff
@@ -2,20 +2,20 @@
 # Rename to .env
 
 # Common config for forge deployment
-RPC_URL="https://rpc-snaxchain-s50q0kjngn.t.conduit.xyz/"
+RPC_URL="https://testnet.snaxchain.io"
 FORGE_ARGS="--legacy"
 
 # Wormhole Core Migrations
 INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
 INIT_CHAIN_ID=43
 INIT_GOV_CHAIN_ID=0x1
 INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
-INIT_EVM_CHAIN_ID=2192
+INIT_EVM_CHAIN_ID=13001
 
 # Bridge Migrations
 BRIDGE_INIT_CHAIN_ID=43
 BRIDGE_INIT_GOV_CHAIN_ID=0x1
 BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
 BRIDGE_INIT_FINALITY=1
-# TODO: This exists. Is this what we should use?
+# I think this is right. It’s what we use on Optimism, and it exists on Snax. . .
```

Comment:
Yes, Snaxchain is an OP Stack chain and therefore would have the same WETH address.